cirq/google/engine/engine.py

```diff
@@ -113,65 +105,59 @@ class Engine:
     Another set of methods return information about programs and jobs that
     have been previously created on the Quantum Engine, as well as metadata
     about available processors:
-        get_calibration
-        get_job
-        get_job_results
-        get_latest_calibration
```

Comment:
Do we want to leave these functions in here for a little while but mark them deprecated?
Otherwise, I think we will probably break some people's code.

They can always delegate to get_processor().function(), right?

Reply:
We've made the decision in this PR to be okay breaking existing users. We could easily support these helpers but unless we really want them as part of the interface I'd rather break people now.

Reply:
Ok, let's make sure to send to the QE communications list to let them know of all of the breaking changes and how to mitigate it.